code/datums/gamemodes/campaign/loadout_items/SOM/pockets.dm

```diff
@@ -30,6 +30,14 @@
 	item_typepath = /obj/item/storage/pouch/grenade/som/standard
 	jobs_supported = list(SOM_SQUAD_MARINE, SOM_SQUAD_CORPSMAN, SOM_SQUAD_VETERAN, SOM_SQUAD_LEADER, SOM_FIELD_COMMANDER)
 
+/datum/loadout_item/r_pocket/som_emp_grenades
+	name = "EMP nades"
```

Comment:
Its nades because the name overflows if its too long. In the case of EMP it probably won't, but then it'll be inconsistant with the other nade pouch names.